scrapers_next/nd/committees.py

```diff
@@ -0,0 +1,90 @@
+from spatula import URL, HtmlListPage, HtmlPage, XPath
+from openstates.models import ScrapeCommittee
+import re
+
+
+class SubcommitteeFound(BaseException):
+    def __init__(self, com_name):
+        super().__init__(
+            f"Scraper has no process for ingesting subcommittee classification: {com_name}"
+        )
+
+
+class CommitteeDetail(HtmlPage):
+    def process_page(self):
```

Comment:
Another comment of mine requests the addition of statutory committees in `CommitteeList`, and provides the implementation details. Fortunately, the code you've written in `CommitteeDetails` still allows your to get all of the members of MOST of the statutory committees, but you will need a way to get "Citizen Members" on pages like [this](https://www.ndlegis.gov/assembly/67-2021/committees/joint/western-interstate-commission-higher-education) and [this](https://www.ndlegis.gov/assembly/67-2021/committees/joint/veterinary-medical-education-program-admissions-committee).

Good news is they should be easy to target with a selector like `citizens = XPath("//div[@class='item-list']//li").match(self.root)`.

For example on how those members should be stored, look to the first linked committee in this comment.
- the first citizen member listed is "Dr. Mark R. Hagerott, North Dakota University System"
- he should be stored with `name` as `"Dr. Mark R. Hagerott"` and `role` as `"Citizen Member (North Dakota University System)"`
   - However, there is not always a role listed. While you may be tempted to split name and role using the fact that they are separated by a comma, you should instead target the role using the fact that (when present) it is always contained within `<em>` element within the list item element. So you should have a conditional that gets and constructs the role detail when that` <em>` element is present within one of those list items.
      - In the case when that role detail is not listed, then their member `"role"` should just be stored as "Citizen Member"

Reply:
added them into com.add_member() is this correct?

Reply:
Yes, and the scraped data looks good for membership now.